src/server/editorServices.ts

```diff
@@ -861,7 +865,7 @@ namespace ts.server {
                 extraFileExtensions: [],

```

Comment:
I think that it makes sense for fshost to provide newLine, although I'm not certain.

Reply:
i think newLine is not part of fsHost but this.host because we would need to get newLine always from this.host to populate onto fsHost (vfs has no newLine concept)

Reply:
But its similar to `useCaseSensitiveFileNames`  which has to be passed to vfs so that it behaves correctly... But will be populated from host.. So i guess if it makes easier, newLine can be on fsHost